packages/cozy-realtime/src/CozyRealtime.spec.js

```diff
@@ -608,5 +608,31 @@ describe('CozyRealtime', () => {
         server.stop(done)
       })
     })
+
+    describe('connection concurrency', () => {
+      it('should not start two concurrent connections', async done => {
+        const server = createSocketServer()
+        const realtime = createRealtime()
+        const handler = jest.fn()
+        realtime.subscribe(event, type, undefined, handler)
+        await realtime.waitForSocketReady()
+
+        // makes the reconnect wait
+        realtime.retryManager.onFailure()
+        realtime.retryManager.onFailure()
+        realtime.retryManager.onFailure()
+        realtime.retryManager.onFailure()
+        realtime.retryManager.onFailure()
+        realtime.onOnline()
+        await sleep(25)
```

Comment:
What is this magic value ?

Reply:
It's a "wait a little for async events to take place"